code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -63,18 +63,16 @@ \subsection{Table of Symbols}
 \\
 $A$ & Area: A part of an object or surface & $\text{m}^{2}$
 \\
-$b$ & Base Width of a Slice: in the x-direction & m
+$\mathbf{b}$ & Base Width of a Slice: in the x-direction & m
 \\
-${C_{den}}$ & Proportionality Constant Denominator: expression used to calculate the denominator of the interslice normal to shear force proportionality constant & N
+${\mathbf{C}_{den}}$ & Proportionality Constant Denominator: expression used to calculate the denominator of the interslice normal to shear force proportionality constant & N
```

Comment:
As for the previous comment, should we make denominators plural, since it is a sequence of denominators?